typing-generator/src/main/kotlin/com/chattriggers/ctjs/typing/Provider.kt

```diff
@@ -234,6 +244,10 @@ class Processor(environment: SymbolProcessorEnvironment) : SymbolProcessor {
         })
 
         indented {
+            if (clazz.classKind == ClassKind.OBJECT) {
+                staticProperties.forEach { buildProperty(it, resolver) }
+                staticFunctions.forEach { buildFunction(it, resolver, omitName = false) }
+            }
```

Comment:
So if the class is `object`, then the static properties will be output for both the instance type and the static type? That makes sense to me, I just want to make sure I'm reading it right

Reply:
Yes exactly